Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -440,6 +456,63 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
             };
         }
 
+        private Operation CreateTrackedChangesKeyChangeOperation(OpenApiMetadataPathsResource openApiMetadataResource)
+        {
+            var responses = OpenApiMetadataDocumentHelper.GetReadOperationResponses(
+                _pathsFactoryNamingStrategy.GetResourceName(openApiMetadataResource, ContentTypeUsage.Readable),
+                true, false, true);
+
+            var parameters = new List<Parameter>
+            {
+                new Parameter {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("offset")},
+                new Parameter {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("limit")},
+                new Parameter {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("MinChangeVersion")},
+                new Parameter {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("MaxChangeVersion")},
+                new Parameter {@ref = OpenApiMetadataDocumentHelper.GetParameterReference("totalCount")}
+            };
+
+            if (_apiSettings.IsFeatureEnabled(ApiFeature.ChangeQueries.GetConfigKeyName()))
+            {
+                responses.Add(
+                    "410",
+                    new Response
+                    {
+                        description =
+                                                        "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
+                    });
+
+                parameters.Add(new Parameter
+                {
+                    name = "Snapshot-Identifier",
+                    @in = "header",
+                    description = "Indicates the Snapshot-Identifier that should be used.",
+                    type = "string",
+                    required = false
+                });
+            }
+
+            return new Operation
+            {
+                tags = new List<string>
+                {
+                    OpenApiMetadataDocumentHelper.GetResourcePluralName(openApiMetadataResource.Resource)
+                        .ToCamelCase()
+                },
+                summary = "Retrieves resources key changes based on change version.",
+                description = "The operation is used to retrieve resources key changes.",
```

Comment:
Change this description to: "This operation is used to retrieve identifying information about resources whose key values have been changed."